### ISSUE TITLE
Tag 1.0.3 & publish to cocoapods?

### ISSUE THREAD

**Issue by LowAmmo** [iziz/libPhoneNumber-iOS#348]
Cocoapods is still pointing to 0.9.15 as the latest version.

Can this pod be tagged at 1.0.3 and re-published to the cocoapods repo so it can be consumed?

-Thanks!

**Comment by LowAmmo** [comment]
Another bump... the Cocoapods version is getting pretty old...

**Comment by paween** [comment]
Yeah. Too bad there's nothing I can do about it because none of us at Google have access to iziz's Cocoapods repo.

**Comment by LowAmmo** [comment]
![Sad Panda](https://c.tenor.com/EWFiAD0KtbkAAAAd/sad-panda.gif)

Since github doesn't have a "sad" reaction option... 😢

**Comment by LowAmmo** [comment]
@iziz - Just bumping again to hopefully update the release in Cocoapods??
